shared/naturalcrit/splitPane/splitPane.jsx

```diff
@@ -83,20 +90,58 @@ const SplitPane = createClass({
 			window.getSelection().removeAllRanges();
 		}
 	},
-*/
+	*/
+
+	setMoveArrows : function(newState) {
+		if(this.state.showMoveArrows != newState){
+			this.setState({
+				showMoveArrows : newState
+			});
+		}
+	},
+
+	renderMoveArrows : function(){
+		if(this.state.showMoveArrows) {
+			return <>
+				<div className='arrow left'
+					style={{ left: this.state.currentDividerPos-4 }}
+					onClick={()=>this.setState({ moveSource: !this.state.moveSource })} >
+					<i className='fas fa-arrow-left' />
+				</div>
+				<div className='arrow right'
+					style={{ left: this.state.currentDividerPos-4 }}
+					onClick={()=>this.setState({ moveBrew: !this.state.moveBrew })} >
+					<i className='fas fa-arrow-right' />
+				</div>
+			</>;
+		}
+	},
+
 	renderDivider : function(){
-		return <div className='divider' onMouseDown={this.handleDown} >
-			<div className='dots'>
-				<i className='fas fa-circle' />
-				<i className='fas fa-circle' />
-				<i className='fas fa-circle' />
+		return <>
+			{this.renderMoveArrows()}
+			<div className='divider' onMouseDown={this.handleDown} >
+				<div className='dots'>
+					<i className='fas fa-circle' />
+					<i className='fas fa-circle' />
+					<i className='fas fa-circle' />
+				</div>
 			</div>
-		</div>;
+		</>;
 	},
 
 	render : function(){
 		return <div className='splitPane' onMouseMove={this.handleMove} onMouseUp={this.handleUp}>
-			<Pane ref='pane1' width={this.state.currentDividerPos}>{this.props.children[0]}</Pane>
+			<Pane
+				ref='pane1'
+				width={this.state.currentDividerPos}
+			>
+				{React.cloneElement(this.props.children[0], {
```

Comment:
Can you clarify to me why we need to have `cloneElement` here?

Reply:
As I understand it, the `this.props.children[0]` is immutable, so we need to clone it in order to modify it to add the extra props.

Reply:
Hm. I looked it up and that seems to be the case. How strange that they would do it that way....

Reply:
I am struck by the thought that we could move the Pagination functions to SplitPane and use a similar method to append the pagination function as a prop to both the Editor and the BrewRenderer.
That probably falls outside the scope of this particular PR, so I'm going to raise the deduplication as a separate issue.

Reply:
Raised as #2198.

Reply:
> Hm. I looked it up and that seems to be the case. How strange that they would do it that way....

I wonder if we couldn't do something like:

```
getDefaultState : function() {
    children: this.props.children
}
```

then modify `this.state.children` and finally pass it as `this.state.children[0]` rather than `this.props.children`.